src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+$square-size: 100px;
+$gap-size: 1px;
+$calendar-padding: 10px;
+$cell-color: #eee;
+$cell-size: 100px;
+$calendar-width: (7 * $cell-size) + $gap-size + $calendar-padding;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  text-align: center;
+  margin: 100px auto;
+  line-height: $cell-size;
+  padding: $calendar-padding;
+  gap: $gap-size;
+  max-width: $calendar-width;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  &__day {
+    height: $cell-size;
+    width: $cell-size;
+    background-color: $cell-color;
+    cursor: pointer;
+
+    transition: transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day:nth-child(#{$i})::before {
+      content: "#{$i}";
+    }
+  }
+
+  $week: 1 mon, 2 tue, 3 wed, 4 thu, 5 fri, 6 say, 7 sun;
+
+  @each $number, $day in $week {
+    &--start-day-#{$day} :first-child {
```

Comment:
this loop doesn't work
![image](https://github.com/mate-academy/layout_calendar/assets/91411692/c49e0266-fba4-461c-8296-ab11357b6437)